devops/terraform/bootstrap.sh

```diff
@@ -1,6 +1,27 @@
 #!/bin/bash
 set -e
 
+# validate environment variables are set
```

Comment:
I think adding the normal header we use elsewhere will accomplish the same thing:

```bash
set -o errexit
set -o pipefail
set -o nounset
```

Reply:
@tamirkamara you are correct, you added this in #2607 so will close the PR and issue.